### ISSUE TITLE
Mismatch issue in dual loop filter unit test

### ISSUE THREAD

**Issue by Cidana-Developers** [OpenVisualCloud/SVT-AV1#546]
Step to reproduce:
`./SvtAv1UnitTests --gtest_filter="*LoopFilter*"`

Console output (Only the failed cases):
```
[ RUN      ] SSE2/LbdDualLoopFilterTest.MatchTestRandomData/0
/Users/yons/work/SVT-AV1/test/DeblockTest.cc:230: Failure
Expected equality of these values:
  0
  err_count_total
    Which is: 5089
Error: Loop8Test6Param, C output doesn't match SIMD loopfilter output. First failed at test case 2
[  FAILED  ] SSE2/LbdDualLoopFilterTest.MatchTestRandomData/0, where GetParam() = (0x10b7d0d80, 0x10b554010, 8) (39 ms)
[ RUN      ] SSE2/LbdDualLoopFilterTest.MatchTestRandomData/1
/Users/yons/work/SVT-AV1/test/DeblockTest.cc:230: Failure
Expected equality of these values:
  0
  err_count_total
    Which is: 5022
Error: Loop8Test6Param, C output doesn't match SIMD loopfilter output. First failed at test case 12
[  FAILED  ] SSE2/LbdDualLoopFilterTest.MatchTestRandomData/1, where GetParam() = (0x10b7d0f70, 0x10b637370, 8) (39 ms)
[ RUN      ] SSE2/LbdDualLoopFilterTest.MatchTestRandomData/2
/Users/yons/work/SVT-AV1/test/DeblockTest.cc:230: Failure
Expected equality of these values:
  0
  err_count_total
    Which is: 74
Error: Loop8Test6Param, C output doesn't match SIMD loopfilter output. First failed at test case 163
[  FAILED  ] SSE2/LbdDualLoopFilterTest.MatchTestRandomData/2, where GetParam() = (0x10b7d0760, 0x10b637cc0, 8) (40 ms)
[----------] 3 tests from SSE2/LbdDualLoopFilterTest (118 ms total)
```
Target functions:
```
aom_lpf_horizontal_4_dual_sse2
aom_lpf_vertical_4_dual_sse2
aom_lpf_horizontal_8_dual_sse2
```

**Comment by ttrigui** [comment]
None of these kernels are being used in the current master. In the process of code cleanup this code will probably be removed.

**Comment by Cidana-Developers** [comment]
Ok, please also remove the unit tests as the functions are removed.

**Comment by hassount** [comment]
@ttrigui can we remove the test to close this?

**Comment by ttrigui** [comment]
> @ttrigui can we remove the test to close this?

issue resolved in commit 7513210
